### PR TITLE
Make gear attributes in stats translatable and reflect if no benefits

### PR DESCRIPTION
**Pull request by ghost**
- Commit 3501dbf addressing my own observation https://github.com/HabitRPG/habitrpg/issues/4522#issuecomment-102659643: Make gear attributes translatable in stats ("STR", "CON", "PER", "INT")
- ~~Commit 2 going a bit further:   New string introduced if gear has no benefits: "Confers no benefit"~~
- replaced with the last commit e487036 to hide colon for items with no stat bonus

I've tried to use the string already present on the site as a newly introduced one, however I'm not sure about the wording (e.g. it's also used with the "No gear: ..." string. So I'm suggesting it with a separate commit so that it can could changed.

Screenshot: A [before / after](http://oi57.tinypic.com/2m2tb40.jpg), B [before / after](http://oi60.tinypic.com/2ursgsn.jpg)

(Reason for upd - screenshot B added)

### CODE REVIEW

File: website/views/shared/profiles/stats.jade [hunk 1]
```diff
@@ -28,7 +28,8 @@ unless mobile
     tr(ng-repeat='(k,v) in profile.items.gear.equipped', ng-init='piece=Content.gear.flat[v]', ng-show='piece')
       td
         strong {{piece.text()}}:&nbsp;
-        span(ng-repeat='stat in ["str","con","per","int"]', ng-show='piece[stat]') {{piece[stat]}} {{stat.toUpperCase()}}&nbsp;
+        span(ng-repeat='stat in ["str","con","per","int"]', ng-show='piece[stat]') {{piece[stat]}} {{env.t(stat)}}&nbsp;
+        span(ng-hide='piece.str || piece.con || piece.per || piece.int')=env.t('noBenefit')
```
Comment (crookedneighbor) [inline website/views/shared/profiles/stats.jade]: I'm not sure I like this bit. Plus, if you aren't wearing a piece of equipment, it looks like this (note the "No Helm" and "No Shield Hand Equipment":

screen shot 2015-06-27 at 9 25 52 pm

Which seems odd to me.

Comment (ghost) [inline website/views/shared/profiles/stats.jade]: Thank you for your comment! I'm not sure myself - that's why I've mentioned
the "no gear:..." case and provided screenshot A. However I believe the
empty space looks even stranger after the colon.

Comment (crookedneighbor) [inline website/views/shared/profiles/stats.jade]: I definitely like "No benefits" better.

Comment (ghost) [inline website/views/shared/profiles/stats.jade]: Thank you! Should I open an issue to wait for other opinions (sorry for not
doing so first)?

Comment (crookedneighbor) [inline website/views/shared/profiles/stats.jade]: The only reason I can think of that you'd want to submit an issue first would be to save you any code work if it was decided that the thing you opened an issue about did not need to be fixed, or needed to be approached in a different way than you were anticipating.

So, I'm fine with a PR. Makes it easier for me to review and think about to see the actual code, and not just the proposed code. :smile:

**Comment by ghost** [comment]
Screenshot of take 2 on New string in stats if gear has no benefits or no gear:
Screenshot

@crookedneighbor could you please tell me if I'm right, git-fu sake, reverting the previous commit and making a new one to the same branch - or should I make a fresh commit from another branch in such cases?

**Comment by ghost** [comment]
Please excuse me for not mentioning it before, but when dealing with this issue I've been rather confused by the [FIXME line](https://github.com/GitHubSphinx/habitrpg/blob/fdbd18a8ed1563c8c61fcb200e3e749324c08a91/website/views/shared/profiles/stats.jade#L24) which opens the part of the code. I haven't understood the wording `// FIXME get translations working before can use this.`

As this part of code is already in use, it gives me an impression that FIXME line mentions translation not exactly of the stats, but seemingly something else (translation system on the whole?). So this PR just introduces translation for the names of stats and a `No benefits` line - leaving the FIXME line in place.

As it's been mentioned now in #5542 as `Translations on stats readout` (_#7 among FIXMEs) I feel I am to ask specifically if this PR fixes it, or maybe unfortunately makes it worse.

**Comment by crookedneighbor** [comment]
This looks good to me. 

@lemoness What do you think about the "No benefits" text? An alternative solution would be to just hide the colon if the item does not give a stat boost. I think that's my preferred way of handling it, but want to hear another opinion. 

@GitHubSphinx I don't think that comment applies anymore. I've [removed it.](https://github.com/HabitRPG/habitrpg/commit/60e92805dd72b3e6ef89bc74a14d5a897c2f45ee)

**Comment by ghost** [comment]
@crookedneighbor oh, thank you! To be honest, I haven't considered removing the colon... It may look nicer for strings like 'No helm', 'No shield', etc. However, I'm not sure if it'll be intuitive why accessories or 'plain' things lack colon and stats as they are just in the same list... So I guess I have no strong opinion (as usual =)

Screenshots of current view vs. suggested changes vs. the solution proposed by @crookedneighbor 

Scr

**Comment by crookedneighbor** [comment]
I like the removing the colon option best, but I'll leave it up to @lemoness to decide.

**Comment by lemoness** [comment]
I like removing the colon! If we get questions, we can always switch to "no
benefits", but I don't see many questions about that even with the colon.

**Comment by ghost** [comment]
@crookedneighbor @lemoness thank you very much for your help and patience! 
I've changed the PR to have the chosen option.

Please excuse me, the PR turned a bit messy:
- [needed] Make gear attributes translatable in stats 3501dbf
- [not needed] c2f0f89 -reverted 
- [not needed] fdbd18a -reverted
- [needed] Make colon is stats hideable e487036

**Comment by ghost** [comment]
@crookedneighbor Just wanted to let you know that I'm leaving in some hours and I'm not sure I will be able to stay in touch. PR seems ready to me, but if there are any additional changes suggested, I'm afraid I won't be able to work on them until I'm back in August. Please excuse me for this. Best regards.

**Comment by crookedneighbor** [comment]
No worries. It looks done to me, but I want to go through it once more to be sure.

**Comment by crookedneighbor** [comment]
Notated toward your next tier.

**Comment by ghost** [comment]
@crookedneighbor thanks for merging =) and thank you very much for attention to small commits!

**Pull request merged**